src/config/power-management.md

```diff
@@ -0,0 +1,29 @@
+# Power Management
+
+Void provides two approaches to power management:
+[acpid(8)](https://man.voidlinux.org/acpid.8) and
+[elogind(8)](https://man.voidlinux.org/elogind.8).
```

Comment:
Feels kind of weird that this paragraph mentions acpid and elogind, but not TLP. They are separate categories of applications, too, so perhaps we could have `## Power management` and `## Power saving` top level headers? Terrible names, just thinking about it.

The takeaway is that I think it makes sense for acpid and elogind to be inside a header.

Reply:
Good points - agreed. i'll see what i can come up with.

Reply:
PR updated. i've used the heading titles you suggested as at least placeholders - let me know of any titles you think might be better. :-)